### ISSUE TITLE
.sbtopts file ignored under Windows?

### ISSUE THREAD

**Issue by alexeyr** [sbt/sbt#4919]
Options set in .sbtopts don't seem to be used under Windows. Looking at https://github.com/sbt/sbt-launcher-package/blob/master/src/universal/bin/sbt.bat, the file isn't accessed.

**Comment by Kesanov** [comment]
So what is the current workaround?

**Comment by melezov** [comment]
Unsure there is any. I use project-specific batch files (e.g. sbt.bat in root folder) instead of the SBT supplied launcher script.

Example project specific runner:
```
@echo off
pushd "%~dp0"

java ^
  -Xms1024M ^
  -Xmx8192M ^
  -Xss1M ^
  -Dsbt.override.build.repos=true ^
  -Dsbt.boot.properties=project/sbt.boot.properties ^
  -Dsbt.repository.config=project/repositories ^
  -jar path/to/sbt-launch.jar ^
  %*

popd
```

BTW, if you're developing using IDEA, it should pickup .sbtopts properly without such hacks.
These are only required for the console.

**Alternatively**, if you're only setting -Xmx and some JVM parameters you can simply use .jvmopts instead of .sbtopts which works on both nux and windoze.

.jvmopts:
```
-Xms1024M
-Xmx8192M
-Xss1M
-Dsbt.override.build.repos=true
-Dsbt.boot.properties=project/sbt.boot.properties
-Dsbt.repository.config=project/repositories
```

**Comment by er1c** [comment]
I believe this was fixed in `1.3.3`
